rst/cartridge_dev.rst

```diff
@@ -1373,7 +1373,7 @@ filename.
 When ``INSTANCE_NAME`` is not provided, ``cartridge`` reads ``cfg`` file and
 starts all defined instances:
 
-.. code-block:: console
+.. code-block:: bash
```

Comment:
```suggestion
..  code-block:: bash
```
Please revise the following lines of this code block so they are aligned with the directive.